### ISSUE TITLE
[BUG]: Errors with Anapnoe UX

### ISSUE THREAD

**Issue by falsonerd** [DominikDoom/a1111-sd-webui-tagcomplete#156]
### Issue:
It's not working properly with Anapnoe UX - https://github.com/anapnoe/stable-diffusion-webui-ux
While it does work, all menus are presenting error... And no "menu related" setting works.

Images at the end....

Edit:
Removed all extensions, and was putting them back one by one, until the error popped up.
Them I proceeded and removed all other, same error still.

#### Env:
![image](https://user-images.githubusercontent.com/50148645/229962371-9ecd32e8-835d-4a8c-981d-fe2b26425490.png)

### Commit
```
4331bdccda003b5bdcfe6f156561631873da3a72
```

### Console Log error:

```
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/dist-packages/gradio/routes.py", line 337, in run_predict
    output = await app.get_blocks().process_api(
  File "/usr/local/lib/python3.9/dist-packages/gradio/blocks.py", line 1018, in process_api
    data = self.postprocess_data(fn_index, result["prediction"], state)
  File "/usr/local/lib/python3.9/dist-packages/gradio/blocks.py", line 947, in postprocess_data
    prediction_value = postprocess_update_dict(
  File "/usr/local/lib/python3.9/dist-packages/gradio/blocks.py", line 371, in postprocess_update_dict
    update_dict = block.get_specific_update(update_dict)
  File "/usr/local/lib/python3.9/dist-packages/gradio/blocks.py", line 257, in get_specific_update
    specific_update = cls.update(**generic_update)
TypeError: update() got an unexpected keyword argument 'multiline'
```

### Images
![image](https://user-images.githubusercontent.com/50148645/229962127-6ae51aaf-951a-41a7-8cb5-1ee81b0321bf.png)
![image](https://user-images.githubusercontent.com/50148645/229962194-e2731a4e-7fb6-4492-8865-165f26890f9a.png)

**Comment by The7ofDiamonds0000** [comment]
I am having [what looks like] a pretty much identical error in the Automatic1111 interface, even seeing that "multiline" thing in the automatic1111 log.  I disabled all extensions and one by one enabled them all and figured out that autocomplete was responsible for the error.  

Notably, I am not using the latest Automatic1111 commit, due to other current errors.  But the one which is by now probably 2-3 weeks old had stopped working with Autocompleted enabled.

**Comment by DominikDoom** [comment]
Sorry for the inconvenience, the multiline argument was wrongly included in the fallback for gradio 3.16.
Due to Gradios poor changelog documentation, I wasn't aware that this argument is also only available in the more recent versions.

**Comment by The7ofDiamonds0000** [comment]
Thanks for the quick resolution.  Love your extension, been so helpful for a lot of stuff!
